scripts/create_input_data_overview.py

```diff
@@ -0,0 +1,76 @@
+# coding: utf-8
+r"""
+Inputs
+------
+in_path : str
+    ``raw/{scalars}.csv``: path to raw scalars
+out_path : str
+    ``results/_tables/{scalars}_technical_and_cost_assumptions.csv``: target path for the table
+
+Outputs
+-------
+.csv
+    Table showing investment cost and efficiency data.
+
+Description
+-----------
+This script creates overview tables of input data such
+that it can be included in a TeX-document.
+"""
+import sys
+
+from oemof_b3 import labels_dict
+from oemof_b3.tools.data_processing import load_b3_scalars
+
+SCENARIO = "Base 2050"
+REGION = "ALL"
+INDEX = ["carrier", "tech", "var_name"]
+VAR_NAMES = {
+    "capacity_cost_overnight": "Overnight cost",
+    "lifetime": "Lifetime",
+    "fixom_cost": "Fix OM cost",
+    "efficiency": "Efficiency",
+}
```

Comment:
For display in results document it would be nice if you added units here.
I also though that you could round "Efficiency", as there are values with 1,2 and 3 decimal places now